src/ripple/app/rdb/backend/RelationalDBInterfaceSqlite.cpp

```diff
@@ -457,14 +470,20 @@ RelationalDBInterfaceSqliteImp::getMinLedgerSeq()
         return ripple::getMinLedgerSeq(*db, TableType::Ledgers);
     }
 
-    /* else use shard databases */
-    std::optional<LedgerIndex> res;
-    iterateLedgerForward(
-        {}, [&](soci::session& session, std::uint32_t shardIndex) {
-            res = ripple::getMinLedgerSeq(session, TableType::Ledgers);
-            return !res;
-        });
-    return res;
+    /* else use shard databases, if available */
```

Comment:
The code is better off without comments that repeat what the next line of code does (here and in many other places in this file). You don't need to change that now (it's a minor nit), but new code should not include such comments.

Reply:
Noted. I'll consider removing these as I dig deeper into refactoring.

Reply:
Ended up removing them when addressing CJ's comment about returning early when transaction tables are disabled. Done